### PR TITLE
Add support for model release level and serial

### LINKED ISSUE #4: Expose model `release_version` and `release_level` in various outputs
See https://github.com/chop-dbhi/data-models/issues/80

### PR DESCRIPTION
**Pull request by bruth**
These are integrated as optional fields since some models may not
require them.

Fix #4

Signed-off-by: Byron Ruth b@devel.io

### CODE REVIEW

**Comment by bruth** [comment]
@aaron0browne Updated to per the https://github.com/chop-dbhi/data-models/pull/81 PR

File: types.go [hunk 1]
```diff
@@ -97,7 +97,7 @@ var FileTypeFields = map[FileType][]string{
 		"comment",
 	},
 
-	ModelsFile: []string{
+	ModelsFile: {
```
Comment (gracecodes144) [inline types.go:100]: Does this type definition need to be changed to include the new `release_level` and `release_serial` fields?

Comment (bruth) [inline types.go:100]: Although it is unclear, these fields are considered required for the file detection. I don't consider these fields to be a hard requirement.

Comment (gracecodes144) [inline types.go:100]: Understood.

**Comment by gracecodes144** [comment]
LGTM

**Pull request merged**